public/js/template.js

```diff
@@ -0,0 +1,52 @@
+import homeView from '../../views/home.hbs';
+import { initEventListeners } from './filter_listeners';
+import { initCalendarEventListener } from './main.js';
+
+let eventsState = [];
+
+export function renderStuff(newFilter) {
+  let filteredEvents = eventsState.filter(event => event[newFilter.lang]);
+  let newDateTo;
+  let newDateFrom;
+  if (newFilter.date_from) {
+    newDateTo = new Date('2018-11-2');
+    newDateFrom = new Date(newFilter.date_from);
+    filteredEvents = filteredEvents.filter(
+      event =>
+        new Date(event.startTime) >= newDateFrom &&
+        new Date(event.startTime) <= newDateTo
+    );
+  } else if (newFilter.categories && !newFilter.date_from) {
+    newDateFrom = new Date();
+    filteredEvents = filteredEvents.filter(
+      event => new Date(event.startTime) > newDateFrom
+    );
+  } else {
+    newDateFrom = new Date();
```

Comment:
`const`

Reply:
Or actually... Maybe just call `new Date()` within the functions as that's what you're doing for the start time anyway...